src/bitcode_lib.cpp

```diff
@@ -9,25 +9,56 @@
            Builtiins-c, or ISPCTarget).
 */
 
+#include <stdlib.h>
```

Comment:
Do the ISPC sources adhere to the LLVM coding style of includes?
https://llvm.org/docs/CodingStandards.html#include-style

Reply:
I am not sure. It looks like almost except system headers place.